descartes_trajectory/test/cart_trajectory_pt.cpp

```diff
@@ -1,160 +0,0 @@
-/*
```

Comment:
You know there is a way to disable a test, by adding DISABLED_ to the test name if I remember correctly.  I do this temporarily when I am debugging.